packages/config/src/options.js

```diff
@@ -43,6 +43,21 @@ export function getNuxtConfig(_options) {
     options.extensions = [options.extensions]
   }
 
+  const styleExtensionDefaults = ['css', 'pcss', 'postcss', 'styl', 'stylus', 'scss', 'sass', 'less']
+
+  if (!options.styleExtensions) {
+    options.styleExtensions = styleExtensionDefaults
+  }
+
+  if (typeof options.styleExtensions === 'function') {
+    // Execute functions with defaults passed in as first argument
+    options.styleExtensions = options.styleExtensions(styleExtensionDefaults)
+  }
+
+  if (typeof options.styleExtensions === 'string') {
```

Comment:
Why supporting such rare types and adding complexity?

Reply:
TBH: I mimicked that behavior from the `extensions` property. :see_no_evil: 

If it's not needed, I can remove it for sure :)